db/models.py

```diff
@@ -17,11 +22,16 @@
 
 
 class Movie(models.Model):
-    title = models.CharField(max_length=255)
+    title = models.CharField(max_length=255, )
```

Comment:
Remove the trailing comma in the `title` field of the `Movie` model for consistency.